release-notes/opensearch-security-analytics-dashboards.release-notes-2.6.0.0.md

```diff
@@ -0,0 +1,37 @@
+## Version 2.6.0.0 2023-02-22
+
+Compatible with OpenSearch 2.6.0
+
+### Maintenance
```

Comment:
We typically include the version bump PR in this section. Would you mind adding it when it becomes available?